src/test/java/com/google/sps/servlets/DBUtilitiesTest.java

```diff
@@ -0,0 +1,98 @@
+package com.google.sps.servlets;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.servlets.DBUtilities;
+import com.google.sps.servlets.Property;
+
+@RunWith(JUnit4.class)
+public final class DBUtilitiesTest {
+  //set up database
+  //call function
+  //assert database
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  private DatastoreService setDatabaseUp() {
+    DatastoreService dataStore = DatastoreServiceFactory.getDatastoreService();
+    
+    Entity originalInteraction = new Entity("Interactions");
+    originalInteraction.setProperty(Property.CORRELATOR, "Person1");
+    originalInteraction.setProperty(Property.FIND_NEAREST_LOCATION, false);
+    originalInteraction.setProperty(Property.GRANTS_LOCATION, false);
+    originalInteraction.setProperty(Property.INTERACTS_WITH_MAP, false);
+    originalInteraction.setProperty(Property.SKIP_TO_CONTENT, false);
+    originalInteraction.setProperty(Property.RETURN_TO_AD, false);
+
+    dataStore.put(originalInteraction);
+
+    return dataStore;
+  }
+
+  @Test
+  public void testIfOnlyOneEntryMade() {
+    DatastoreService dataStore = setDatabaseUp();
+
+    DBUtilities.setToTrue("Person1", Property.FIND_NEAREST_LOCATION);
+
+    assertEquals(1, dataStore.prepare(new Query(DBUtilities.INTERACTION_TABLE)).countEntities(withLimit(10)));
+  }
+
+  @Test
+  public void checkIfPropertyUpdatedCorrectly() {
+    DatastoreService datastore = setDatabaseUp();
+    
+    DBUtilities.setToTrue("Person1", Property.GRANTS_LOCATION);
```

Comment:
maybe specify that GRANTS_LOCATION is only an example of a property that you can test with this particular unit test, but that it's applicable for ALL property tests

Reply:
okay thanks